player.go

```diff
@@ -19,7 +19,8 @@ import (
 	"time"
 )
 
-// A Player is a sound player.
+// Player is a PCM (pulse-code modulation) audio player. It implements io.Writer, use Write method
```

Comment:
I don't think this is idiomatic. Standard library prefers "Type..." instead of "A Type...", there are a few exceptions. Also, see https://golang.org/doc/effective_go.html#commentary